cfgov/cfgov/settings/base.py

```diff
@@ -710,21 +710,6 @@
     "/django-admin",
 )
 
-
-# Django 3.2 Baseline required settings
-# exempt beta from CSRF settings until it's converted to https
-SECURE_REFERRER_POLICY = "same-origin"  # 1
-SESSION_COOKIE_SAMESITE = "Strict"  # 3
-X_FRAME_OPTIONS = "SAMEORIGIN"  # 13
-
-if DEPLOY_ENVIRONMENT:
```

Comment:
This is a good catch, since https://github.com/cfpb/consumerfinance.gov/commit/a1ba4a37fb92ffdaadcafd316043cd33dad27de7?diff=unified&w=0#diff-a81875b2fe9c4c7ccd4061f42e87e7a281ede3c676a3f92db187c20c8e6905b1R126 added a fallback local setting for DEPLOY_ENVIRONMENT, so it's set everywhere.